pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -141,6 +143,77 @@ class TestState(Enum):
 }"""
 
 
+class EpochConfigOverrides:
```

Comment:
nit: feel free to keep it if it's more your style and you want it that way, but why a class? The only place it's used is in the single statement `EpochConfigOverrides(...).override(...)`, which to me is an indication you could just keep it simpler and write this as a normal function